doc/release_notes_0.2.rst

```diff
@@ -17,8 +17,63 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+VT-x, VT-d
+================
+Based on Intel VT-x virtualization technology, ACRN emulates virtual CPU with core partition and simple schedule.
+VT-d stands for Intel VT for Directed IO, providing capabilities to assign I/O devices to VMs and extending the protection and isolation properties of VMs for I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+================
+ACRN hypervisor supported APIC-V/EPT/IOAPIC/LAPIC functionality.
+
+Ethernet
+================
+ACRN hypervisor supported Ethernet functionality. Ethernet Mediator is executed in the Service OS and  provided packet forwarding between the physical networking devices (Ethernet, Wi-Fi, etc.) and virtual devices in the Guest VMs (also called "User OS"). Virtual Ethernet device could be shared by 1-2 Linux, Android, and Service OS guests for regular (i.e. non-AVB) traffic. All hypervisor para-virtualized I/O is implemented using the VirtIO specification Ethernet pass-through.
```

Comment:
change ``supported`` to ``supports virtualized``
change ``provided`` to ``provides``
I think you should delete ``1-2``

Also, break up this paragraph to lines with < 80 characters.